pjsip4net/Calls/Call.cs

```diff
@@ -277,7 +277,7 @@ public void Transfer(string destinationUri)
             GuardDisposed();
             Helper.GuardNotNullStr(destinationUri);
             Helper.GuardIsTrue(new SipUriParser(destinationUri).IsValid);
-            if (_mediaSession.IsActive)
+            if (_mediaSession.IsActive || this._mediaSession.MediaState == CallMediaState.None)

```

Comment:
Okay I see what you mean, but then the 'Theory' (sequential) in the above mentioned test is also superfluous